src/main/java/frc/trigon/robot/subsystems/pitcher/PitcherConstants.java

```diff
@@ -40,23 +40,23 @@ public class PitcherConstants {
     private static final boolean FOLLOWER_OPPOSES_MASTER = false;
     private static final NeutralModeValue NEUTRAL_MODE_VALUE = NeutralModeValue.Brake;
     private static final double
-            P = RobotHardwareStats.isSimulation() ? 200 : 0,
+            P = RobotHardwareStats.isSimulation() ? 100 : 0,
             I = RobotHardwareStats.isSimulation() ? 0 : 0,
-            D = RobotHardwareStats.isSimulation() ? 0 : 0,
-            KS = RobotHardwareStats.isSimulation() ? 0.46583 : 0,
-            KV = RobotHardwareStats.isSimulation() ? 19.611 : 0,
-            KA = RobotHardwareStats.isSimulation() ? 0.48644 : 0,
+            D = RobotHardwareStats.isSimulation() ? 20 : 0,
+            KS = RobotHardwareStats.isSimulation() ? 0.2 : 0,
+            KV = RobotHardwareStats.isSimulation() ? 32 : 0,
```

Comment:
how did you calibrate this? It doesn't look right to me.

Reply:
It's all fine